configure.ac

```
@@ -5476,6 +5476,21 @@ then
   [Define if you have struct stat.st_mtimensec])
 fi
 
+# checking for glibc extensions to langinfo.h
+#  _NL_TIME_FIRST_WEEKDAY isn't #define'd, so test it directly
+AC_MSG_CHECKING(for _NL_TIME_FIRST_WEEKDAY)
+AC_CACHE_CHECK(ac_cv_have__nl_time_first_weekday,
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <langinfo.h>]], [[
+return !!nl_langinfo(_NL_TIME_FIRST_WEEKDAY);
+]])],
+  [ac_cv_have__nl_time_first_weekday=yes],[ac_cv_have__nl_time_first_weekday=no]))
```

AC_CACHE_CHECK cmbines AC_MSG_CHECKING and AC_CACHE_VAL:

```suggestion
AC_CACHE_CHECK([for _NL_TIME_FIRST_WEEKDAY], [ac_cv_have__nl_time_first_weekday], [
  AC_COMPILE_IFELSE([AC_LANG_PROGRAM([#include <langinfo.h>], [
      return !!nl_langinfo(_NL_TIME_FIRST_WEEKDAY)
    ])], [ac_cv_have__nl_time_first_weekday=yes], [ac_cv_have__nl_time_first_weekday=no])
])
```